pkg/operator/config/utils.go

```diff
@@ -60,6 +64,66 @@ func trimSlice(s []interface{}) []interface{} {
 	return res
 }
 
+// intoStages converts the a yaml slice of stages into a Jsonnet array.
+func intoStages(i []interface{}) (interface{}, error) {
+	text, ok := i[0].(string)
```

Comment:
Do we need to protect against 0 length arrays?

Reply:
Nah, jsonnet will prevent that function from being called if you don't have an argument matching for the list of params passed to `vm.NativeFunction`.